src/Extension.ts

```diff
@@ -70,5 +70,34 @@ export class Extension {
 				}
 			)
 		);
+
+		this.dispose.track(
+			vscode.commands.registerCommand(
+				"hediet.vscode-drawio.newDiagram", () => {
+					const options: vscode.SaveDialogOptions = {
+						saveLabel: 'Create',
+						filters: {
+							'Diagrams': ['drawio']
+						}
+					};
+
+					vscode.window.showSaveDialog(options).then(fileUri => {
+						if (fileUri) {
+							var fs = require('fs');
```

Comment:
You should import this a the top of the file with `import` to get typesafety!
Also, you shouldn't use the `fs` module at all in a vs code extension. Use `import { workspace } from "vscode";` instead and then `workspace.fs` I think.
This way it will also work for remote scenarios!

Reply:
I never know about this before that we can use fs in workspace which also support the remote scenarios. Great thanks!